MapboxSearch.podspec

```diff
@@ -24,6 +24,6 @@ Some iOS platform specifics applies.
 
   s.vendored_frameworks = "**/#{s.name}.xcframework"
 
-  s.dependency "MapboxCoreSearch", '2.5.0'
+  s.dependency "MapboxCoreSearch", '2.5.1'
```

Comment:
Was this pull created in advanced? [I don't see MapboxCoreSearch v2.5.1 release in CocoaPods](https://github.com/CocoaPods/Specs/tree/master/Specs/6/8/5/MapboxCoreSearch).

Reply:
Yes

Reply:
Yes pod has been published